src/webgpu/api/validation/gpu_external_texture_expiration.spec.ts

```diff
@@ -187,35 +196,28 @@ g.test('import_and_use_in_different_task')
     });
 
     await waitForNextTask(() => {
-      // Enter in another task scope, previous GPUExternalTexture should be expired.
-      t.submitCommandBuffer(bindGroup, false);
+      // Enter in another task scope. For GPUExternalTexture imported from WebCodec,
+      // it shouldn't be expired because VideoFrame is not 'closed'.
+      // For GPUExternalTexutre imported from HTMLVideoElement, it should be expired.
+      t.submitCommandBuffer(bindGroup, sourceType === 'VideoFrame' ? true : false);
     });
   });
 
 g.test('use_import_to_refresh')
   .desc(
     `
     Tests that in the different task scope, imported GPUExternalTexture
-    again on the same source frame should return the same GPUExternalTexture
-    object and refresh it.
+    again on the same HTMLVideoElement should return active GPUExternalTexture.
     `
   )
-  .params(u =>
-    u //
-      .combine('sourceType', ['VideoElement', 'VideoFrame'] as const)
-  )
```

Comment:
I think it makes sense to bring this back now?

Reply:
A bit hesitate on this.  
First, Refresh seems specific to `GPUExternalTexture` from  `HTMLVideoElement`. Second, the test case  "import_and_use_in_different_task" ([1], https://github.com/gpuweb/cts/pull/2765/files#diff-8e3cd7410d080fa936b8cbe9c85289ca6de0548752bcf72432096d0ed8f5d9b0R162) has already covered the case that `GPUExternalTexture` from `VideoFrame` is valid in other task scope if source frame is not closed

Reply:
I was thinking we need a test that could trigger caching for VideoFrame, where it could return the same GPUExternalTexture twice for the same VideoFrame. But it looks like that's covered by `import_multiple_times_in_same_task_scope`. The two tests are slightly different but between `import_multiple_times_in_same_task_scope` and `import_and_use_in_different_task` I think the coverage is complete. Thanks for explaining!